filecoin-proofs/src/bin/paramcache.rs

```diff
@@ -84,16 +87,23 @@ pub fn main() {
 
     let test_only: bool = matches.is_present("test-only");
 
-    cache_porep_params(PoRepConfig(SectorSize::OneKiB, PoRepProofPartitions::Two));
-    cache_post_params(PoStConfig(SectorSize::OneKiB, PoStProofPartitions::One));
+    cache_porep_params(PoRepConfig(
+        SectorSize(TEST_SECTOR_SIZE),
```

Comment:
Is there a reason that `{TEST|LIVE}_SECTOR_SIZE` are not of type `SectorSize` already?